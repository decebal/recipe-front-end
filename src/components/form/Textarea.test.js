import React from 'react';
import renderer from 'react-test-renderer';
import Textarea from './Textarea';


test('Textarea renders correctly', () => {
  const tree = renderer
    .create(<Textarea
      id="invalid-uuid-4"
      label="textarea"
      value="Text"
      type="text"
      onChange={() => {}}
    />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
