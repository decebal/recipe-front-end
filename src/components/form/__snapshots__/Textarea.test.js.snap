// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea renders correctly 1`] = `
<div
  className="sc-bxivhb gCIOfD"
>
  <label
    className="sc-bdVaJa bNfbPK"
    htmlFor="invalid-uuid-4"
  >
    textarea
  </label>
  <textarea
    className="sc-htpNat bPsheZ"
    id="invalid-uuid-4"
    onChange={[Function]}
    type="text"
  >
    Text
  </textarea>
</div>
`;
